src/seedu/addressbook/data/person/Person.java

```diff
@@ -8,7 +8,7 @@
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated.
  */
-public class Person implements ReadOnlyPerson {
+public class Person implements ReadOnlyPerson, Comparable<Person> {
```

Comment:
Would it be better if we define a Comparator, since this gives more flexibility in the ordering than implementing a Comparable interface?

Reply:
In the scope of this PR, I think Comparable would suffice as the current sort command only aim to order the list of persons by names. Comparator would be necessary only if Tzer Bin wants to expand sort command to feature other ordering sequences.